src/Compilers/CSharp/Portable/Symbols/Source/SourceConstructorSymbol.cs

```diff
@@ -35,14 +34,16 @@ internal sealed class SourceConstructorSymbol : SourceConstructorSymbolBase
              base(containingType, location, syntax, SyntaxFacts.HasYieldOperations(syntax))
         {
             bool hasBlockBody = syntax.Body != null;
-            _isExpressionBodied = !hasBlockBody && syntax.ExpressionBody != null;
-            bool hasBody = hasBlockBody || _isExpressionBodied;
+            bool isExpressionBodied = !hasBlockBody && syntax.ExpressionBody != null;
+            bool hasBody = hasBlockBody || isExpressionBodied;
```

Comment:
NIt: we have all sorts of logic for computing these values in different places.  We also name the variable inconsistently all over the place (hasBody sometimes means 'has any body' and sometimes means 'has block body').  It woudl be good to make this all consistent.